src/components/Popups/ReplyField/ItemList.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { SyntheticEvent } from 'react';
+import DatalistItem from 'box-ui-elements/es/components/datalist-item';
+import noop from 'lodash/noop';
+import { Collaborator } from '../../../@types';
+
+export type Props = {
+    activeItemIndex?: number;
+    filterString?: string;
+    itemRow?: JSX.Element;
+    items: Collaborator[];
+    onSelect: (index: number, event: React.SyntheticEvent) => void;
+    setActiveItemIndex?: (index: number) => void;
+};
+
+const DefaultItemRow = ({ item, ...rest }: Collaborator): JSX.Element | null =>
+    item?.name ? (
+        <DatalistItem {...rest}>
+            <div className="ba-PopupList-name">{item.name}</div>
+            {'email' in item && <div className="ba-PopupList-email">{item.email}</div>}
+        </DatalistItem>
+    ) : null;
+
+const ItemList = ({
+    activeItemIndex = 0,
+    itemRow = <DefaultItemRow {...({} as Collaborator)} />,
```

Comment:
We should avoid casting incompatible data to arbitrary types. Can we make these props optional in the component?